python/tutorials/06-fused-attention.py

```diff
@@ -238,7 +238,7 @@ def forward(ctx, q, k, v, sm_scale):
     @staticmethod
     def backward(ctx, do):
         if torch.version.hip is not None:
-            BLOCK = 64
+            BLOCK = 32
```

Comment:
this is the reason, why I have mismatches =_=

Reply:
@vgokhale FYI